LondonBritishLibrary/orient/BLorient13156.xml

```diff
@@ -54,56 +115,134 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </decoNote>
                             <decoNote type="miniature" xml:id="d8">
                                 <locus target="#144v"/>
-                                <desc>St Onuphrius</desc>
+                                <desc><ref type="authFile" corresp="AT1035HolyMan">Miniature</ref> depicting 
+                                    <persName ref="PRS7665Onuphrio">St Onuphrius</persName>.</desc>
+                                <q xml:lang="gez">ሥዕለ፡ አባ፡ ወንፍር፡</q>
+                            </decoNote>
+                            <decoNote type="diagram" xml:id="d9">
+                                <locus target="#23v"/>
+                                <desc>Symbolic signs referring to Christ on the margins on <ref target="#23v"/> and on various pages of the codex.</desc>
                             </decoNote>
                         </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="1v" to="2r"/>
+                                    <desc type="GuestText">Prayer probably taken from <ref type="work" corresp="LIT2000Mazmur">Psalm 10</ref>.</desc>
+                                    <q xml:lang="gez">በእግዚአብሔር፡ ተወከልኩ፡ እፎ፡ ትብልዋ፡ ለነፍስየ፡ </q>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="2r" to="3r"/>
+                                    <desc type="GuestText"><ref type="work" corresp="LIT3123Calend">A calendar for the commemoration of saints</ref>.</desc>
+                                    <q xml:lang="gez">አመ፡ ፩በመስከረም፡ በርተሎምዮስ፡ ሀዋርያ፡ አመ፲፪፡ 
+                                        በጥም<supplied reason="undefined" resp="PRS8999Strelcyn">ቅ</supplied>ት፡ ማቴዎስ፡ <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="a3">
+                                    <locus from="159r" to="159v"/>
+                                    <desc type="GuestText">Magical prayer and a prescription against snake bite.</desc>
+                                    <q xml:lang="gez">ምንት፡ ውእቱ፡ ፈውስ፡ ለከይሲ፡ መልአከ፡ ቀረጽ፡ ተቀረጽ፡ ወአጥዐጥ፡ ፈውሰ፡ ለከይሲ፡ አኀዝኩ፡ ቀረጽ፡ ወዐጥዐጥ፡ በዕፀ፡ መስቀሉ፡ 
+                                         ለክርስቶስ፡ <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#160v"/>
+                                    <desc type="Unclear">Scribbles</desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">Note written in Italianate Latin on the inside of the front cover.</desc>
+                                </item>
+                                <item xml:id="e3">
```

Comment:
It would be good to add `<q xml:lang="la">` (and possible a second one for Italian) to make this filterable